tokio/src/runtime/scheduler/multi_thread/worker.rs

```diff
@@ -287,7 +287,7 @@ pub(super) fn create(
             remotes: remotes.into_boxed_slice(),
             inject,
             idle,
-            owned: OwnedTasks::new(),
+            owned: OwnedTasks::new(16),
```

Comment:
how was 16 chosen as the value for this? i see that you ran the benchmarks on a Xeon Gold 6133, which has 16 CPU cores...would it be more correct to determine this value based on `num_cpus` to ensure that it's always the number of CPU cores, regardless of which CPU is in use?

Reply:
I run on a `Intel(R) Xeon(R) Gold 6133 CPU @ 2.50GHz.` , which has 8 CPU cores. The reason I choose 16 is for performance testing, and this size can be discussed further.

Actually I decided to make this size twice the number of CPU cores, because the number of worker threads in tokio is the number of CPU core. Meanwhile, I want it to be a integer power of 2.

So we could just set it to a big enough vlaue for most platform, like 64, and let people to configure this. But it might cause CPU cache problem.

Or, set it to an integer power of two that is at least twice the number of CPU cores.